### DISCUSSION TITLE
Walkthrough: NFT panoply (review grid)

### DISCUSSION THREAD

**Discussion by razgraf** [sablier-labs/evm-monorepo#520] [Show and tell]
Now that we have access to a solution that generates NFTs (`shell/generate-svg-panoply.sh`) for visual checkup and regression testing, I've coded a small HTML/JS script that helps organize them in a grid for quick review.

You can access the code in this [gist](https://gist.github.com/razgraf/5d6064f0be745764f019923bb668a0a3). To run it, simply paste it into a `panoply.html` file and open that file in your browser. Click on the input, select items from the `out-svg` folder and voila. I've purposely written it in vanilla javascript with an input selector rather than `fs` to make it as plug-n-play as possible.

<img width="600" alt="nft-gallery-96d6faf8baa6fbf690ea89d4252ea727" src="https://github.com/sablier-labs/v2-core/assets/8782666/117021eb-3c49-45dd-bc67-25364ed17d5f">

**Comment by PaulRBerg** [comment]
Cc @maxdesalle this script will be super helpful for pitching our NFT SVGs to NFT platforms (when talking to them about uncaching our SVGs).
